src/applications/disability-benefits/526EZ/pages/treatmentAddress.js

```diff
@@ -0,0 +1,48 @@
+import { get } from 'lodash';
+
+import  {
+  MILITARY_CITIES,
+  MILITARY_STATE_VALUES,
+  USA,
+} from '../constants';
+
+function validateMilitaryCity(errors, city, formData, schema, messages, index) {
+  const isMilitaryState = MILITARY_STATE_VALUES.includes(
+    get(formData, `treatments[${index}].treatmentCenterAddress.state`, '')
```

Comment:
Did you verify this `index` is for `treatments` and not for `disabilities`? Should be able to do that by making two treatment centers where one has a domestic address and another has a military address.

Reply:
Yep, verified by doing exactly that. Also, by logging out the index property while adding multiple facilities to the same disability.